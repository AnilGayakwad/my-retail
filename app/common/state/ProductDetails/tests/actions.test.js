
import {
  productDetailsRequestSucceeded,
  productDetailsRequestFailed,
} from '../actions';


describe('productDetails actions', () => {
  describe('productDetailsRequestSucceeded', () => {
    it('should have PRODUCT_DETAILS_REQUEST_SUCCEEDED type', () => {
      const registries = {
        pass: 'pass',
      };
      expect(productDetailsRequestSucceeded(registries, '123')).toMatchSnapshot();
    });
  });

  describe('productDetailsRequestFailed', () => {
    it('should have PRODUCT_DETAILS_REQUEST_FAILED type', () => {
      expect(productDetailsRequestFailed(new Error('failed'), '123')).toMatchSnapshot();
    });
  });
});
