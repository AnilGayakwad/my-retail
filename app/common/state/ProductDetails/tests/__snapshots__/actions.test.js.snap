// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`productDetails actions productDetailsRequestFailed should have PRODUCT_DETAILS_REQUEST_FAILED type 1`] = `
Object {
  "error": true,
  "meta": Object {
    "productId": "123",
  },
  "payload": Object {
    "error": [Error: failed],
  },
  "type": "app/ProductDetailsPage/PRODUCT_DETAILS_REQUEST_FAILED",
}
`;

exports[`productDetails actions productDetailsRequestSucceeded should have PRODUCT_DETAILS_REQUEST_SUCCEEDED type 1`] = `
Object {
  "meta": Object {
    "productId": "123",
  },
  "payload": Object {
    "result": Object {
      "pass": "pass",
    },
  },
  "type": "app/ProductDetailsPage/PRODUCT_DETAILS_REQUEST_SUCCEEDED",
}
`;
